JOSESwift/Sources/RSAKeyCodable.swift

```diff
@@ -119,7 +119,7 @@ extension RSAPrivateKey: Decodable {
 
         // Other common parameters are optional.
         var parameters: [String: String] = [:]
-        for key in commonParameters.allKeys {
+        for key in commonParameters.allKeys where key != .X509CertificateChain {
```

Comment:
It would be awesome if you could also ignore the keyOperations parameter here and in the public key decoding.

```suggestion
        for key in commonParameters.allKeys where key != .X509CertificateChain && key != .keyOperations {
```